README.md

```diff
@@ -767,6 +767,27 @@ var rows = [
 ];
 worksheet.addRows(rows);
 
+// Insert a couple of Rows by key-value, shifting down rows every time
```

Comment:
could you please add the same thing to the chinese readme? No need to translate it for now

Reply:
Done